distros/noetic/flatland-server/yaml_preprocessor.patch

```diff
@@ -0,0 +1,22 @@
+--- a/include/flatland_server/yaml_preprocessor.h
++++ b/include/flatland_server/yaml_preprocessor.h
+@@ -47,12 +47,15 @@
+ #ifndef FLATLAND_SERVER_YAML_PREPROCESSOR_H
+ #define FLATLAND_SERVER_YAML_PREPROCESSOR_H
+
++extern "C"
++{
++    #include <lua.h>
++    #include <lualib.h>
++    #include <lauxlib.h>
++}
```

Comment:
Why isn't this necessary on Ubuntu? Does the Ubuntu package really compile Lua as C++?

Reply:
I'm not sure about Ubuntu package, but Nix derivation seems to compile to C
https://github.com/NixOS/nixpkgs/blob/898cb2064b6e98b8c5499f37e81adbdf2925f7c5/pkgs/development/interpreters/lua-5/interpreter.nix#L85